src/server/node_manager.c

```diff
@@ -6319,25 +6481,13 @@ set_nodes(void *pobj, int objtype, char *execvnod_in, char **execvnod_out, char
 		if (execvnod == NULL)
 			return PBSE_BADNODESPEC;
 
-		if (!strlen(execvnod))
+		if (!strlen(execvnod)) {
+			send_nodestat();
```

Comment:
Please describe the flow over here. That the job is being rejected, but should run in the next few cycles by which time the stat would have been answered and populated.